app/controllers/api/v1/tasks_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::TasksController < Api::V1::ApplicationController
+  def index
+    tasks = Task.all.
+      ransack(ransack_params).
+      result.
+      page(page).
```

Comment:
Move the dot to the next line.

Reply:
Model
  .some_scope
  .another_scope
  .some_method

Reply:
Okay. It is company convention? 
In style guide i can not find the best solution
https://github.com/rubocop-hq/ruby-style-guide#leading-dot-in-multi-line-chains

Reply:
There are two approaches to this in the ruby community - `leading .` and `tailing .`. We usually stick to the leading one.
https://github.com/rubocop-hq/ruby-style-guide#leading-

Reply:
Okay, thanks